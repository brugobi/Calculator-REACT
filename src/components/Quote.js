import React from 'react';

function Quote() {
  return (
    <div>
      <p>
        &quot;Why Do We Fall, Sir? So That We Can Learn To Pick Ourselves Up.&quot; -
        <i>Batman Begins</i>
      </p>
    </div>
  );
}

export default Quote;
