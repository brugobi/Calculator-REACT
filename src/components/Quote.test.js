import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

describe('Quote component', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<Quote />);
    expect(tree).toMatchSnapshot();
  });
});
