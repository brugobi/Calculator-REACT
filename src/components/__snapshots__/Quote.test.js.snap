// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component matches the snapshot 1`] = `
<div>
  <p>
    "Why Do We Fall, Sir? So That We Can Learn To Pick Ourselves Up." -
    <i>
      Batman Begins
    </i>
  </p>
</div>
`;
